### DISCUSSION TITLE
fi_info -l missing verbs driver

### DISCUSSION THREAD

**Discussion by brianjmurrell** [ofiwg/libfabric#7332] [Q&A]
Occasionally we will have an interface where `fi_info -d <iface> -l` will be missing the verbs driver.

What causes this, and more importantly, what steps can I take to rectify it?

**Comment by shefty** [comment]
Are the ports on the verbs device active?  You may need to run ibv_devinfo to check the state.  If this is an infiniband device, is the SM running?
